pom.xml

```diff
@@ -2074,6 +2074,9 @@
                                 <include>*/README.md</include>
                                 <include>docker/playground/README.md</include>
                             </includes>
+                            <excludes>
+                                <exclude>docs/kyuubi/**/*.md</exclude>
```

Comment:
are you supposing that the virtual env is named "kyuubi"?

Reply:
I have changed the path to be more general.